consensus/poet/core/sawtooth_poet/poet_consensus/poet_config_view.py

```diff
@@ -139,7 +139,8 @@ def enclave_module_name(self):
                     name='sawtooth.poet.enclave_module_name',
                     value_type=str,
                     default_value=PoetConfigView._ENCLAVE_MODULE_NAME_,
-                    validate_function=lambda value: len(value) > 0)
+                    # function should return true if value is nonempty
+                    validate_function=lambda value: value)
```

Comment:
Ha, you thought it would be that easy?
```
************* Module sawtooth_poet.poet_consensus.poet_config_view
W:143,38: Lambda may not be necessary (unnecessary-lambda)
```